File: hw/elbay/u3/hw25TemplateMethod/hw25TemplateMethod/java/src/main/java/model/SaladDish.java
package model;

public class SaladDish extends MealPreparation {
    @Override
    public void cook() {
        System.out.println("Chopping vegetables and mixing dressing");
    }

    @Override
    public void addGarnish() {
        System.out.println("Adding croutons and sesame seeds");
    }
}

File: hw/elbay/u3/hw25TemplateMethod/hw25TemplateMethod/js/model/SaladDish.js
const MealPreparation = require("./MealPreparation");

class SaladDish extends MealPreparation {
    cook() {
        console.log("Chopping vegetables and mixing dressing");
    }

    addGarnish() {
        console.log("Adding croutons and sesame seeds");
    }
}

module.exports = SaladDish;
